dspace-oai/src/main/java/org/dspace/xoai/services/impl/xoai/DSpaceRepositoryConfiguration.java

```diff
@@ -97,7 +97,7 @@ public String getBaseUrl() {
 
     @Override
     public DeleteMethod getDeleteMethod() {
-        return DeleteMethod.TRANSIENT;
+        return DeleteMethod.PERSISTENT;
```

Comment:
I don't think this PR provides "persistent" delete functionality.  For that to occur, we'd have to be keeping a "tombstone" sort of record for deleted items... 
See https://www.openarchives.org/OAI/openarchivesprotocol.html#DeletedRecords

When I test this PR, it works, but a GetRecord on a deleted item doesn't provide a tombstone, it just says the record is not found.

So, I think this should still be set as "TRANSIENT", unless I'm misunderstanding something in this PR.

Reply:
Sorry about that! Updated the value of delete method.

https://github.com/DSpace/DSpace/pull/2856/commits/d3b127e8cb14fec4318e1085c50f29c93ded0a05

Reply:
@tdonohue @AndrewZWood the purpose to create a solrdoc representing the deleted item in the SOLR oai core is to implement the PERSISTENT level for deletion, otherwise it would be enough to just drop the solr document.

As withdrawn item are shown as deleted oai record, we should investigate why this is not the case for this deleted items. I guess that the solr document must have some extra fields that are used by default by the xoai to make query.

Reply:
To clarify, I'm OK with us attempting to support `PERSISTENT` level of deletion in this PR.  But, currently, the PR works well but only supports `TRANSIENT` deletion.

@AndrewZWood : It looks like @abollini added notes in a comment below about how he thinks `PERSISTENT` deletion might be supported.  If you have time to try it out, we could see if that support is possible.  Otherwise, I'm also OK with merging this PR as-is, since we do not require that this PR support `PERSISTENT` deletion